common/src/main/java/com/wynntils/features/commands/CustomCommandKeybindsFeature.java

```diff
@@ -96,6 +98,8 @@ public class CustomCommandKeybindsFeature extends Feature {
             () -> this.executeKeybind(keybindCommand6.get(), commandType6.get()));
 
     private void executeKeybind(String keybindCommand, CommandType commandType) {
+        Models.WynnLanguage.setSelectedLanguage(WynnLanguage.DEFAULT);
```

Comment:
This bugs me. We got to find a solution to not do this. 

I need to dig deeper into the patch to fully understand why this is here and how to properly resolve it. But it got to go.

Reply:
Yeah bolyai pointed it out too and I don't like it either but it was just a slight improvement over legacy behaviour.
In legacy, it preserves your selected language after sending a message, so this just resets it when the chat screen is opened from a feature that won't want the chat to be translated.

Easy fix is to simply not preserve the language but then if you wanted to send repeated messages in one of the languages, you'd have to click the button each time unless you used the brackets system.

Reply:
So what you are saying is that you really want to catch when the chat edit line appears, and restore the previous language then, but you cannot do that, so you always restore the previous language, and then cancel that for known places where this does not make sense? Did I get it right?

Reply:
So you send a message and once you hit enter, whatever alphabet you last had selected is what the Model remembers so that next time the chat screen is opened it will automatically select that alphabet so that you can continue typing in the alphabet you last were. That's just how legacy behaved. 

But in certain circumstances, such as executing a custom command, or the trade market auto opening chat, you likely don't want that to be typing in anything other than the default alphabet, so it resets to default. I also wanted it to reset to default when you hit / to open the chat screen with / already entered but couldn't figure out how.

I only had it remember as legacy had it and this is a port so wanted to make it function the same but with improvements, but I'm not sure if it's actually worth having as even if you meant to send in a different alphabet, people can still see your message.